common/basic-auth/base/kflogin-virtual-service.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: networking.istio.io/v1alpha3
+kind: VirtualService
+metadata:
+  name: gatekeeper
+spec:
+  gateways:
+  - kubeflow-gateway
+  hosts:
+  - '*'
+  http:
+  - match:
+    - uri:
+        prefix: /kflogin
+    rewrite:
+      uri: /kflogin
+    route:
+    - destination:
+        host: basic-auth.$(service-namespace)
```

Comment:
from the doc it recommends full name: [link](https://github.com/istio/api/blob/6b8d1849e7f44ef523b4442af69b57ddc960d38b/networking/v1alpha3/virtual_service.proto#L330)

could you add a param `clusterDomain` and make this host name `basic-auth.$(service-namespace).svc.$(clusterDomain)` instead?

Reply:
Fixed it :)